Source/WTF/wtf/RunLoop.h

```diff
@@ -185,7 +185,7 @@ class WTF_CAPABILITY("is current") RunLoop final : public GuaranteedSerialFuncti
         template <typename TimerFiredClass>
         requires (WTF::HasRefPtrMemberFunctions<TimerFiredClass>::value)
         Timer(Ref<RunLoop>&& runLoop, TimerFiredClass* object, void (TimerFiredClass::*function)())
-            : Timer(WTFMove(runLoop), [object, function] {
+            : Timer(WTFMove(runLoop), [object, function] SUPPRESS_UNCOUNTED_LAMBDA_CAPTURE {
```

Comment:
I think we need a comment here explaining the expected lifetime model: The Timer's owner is expected to cancel the Timer in its destructor.

Reply:
Sure, added.